src/Command/CommandHandlerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Nihilus\CQRSBundle\Command;
+
+use Nihilus\CommandHandlerInterface as NihilusCommandHandlerInterface;
+
+interface CommandHandlerInterface extends NihilusCommandHandlerInterface
+{
+    public static function getClassHandled(): string;
```

Comment:
getHandledClass instead ?

But actually I think getClass would be enough. What do you think about that ?

Reply:
j'avais peur que getClass ne pas soit pas suffisament explicite. en revanche je suis ok pour getHandledClass si c'est plus logique dans la langue de shakespeare

Reply:
fait